adapter/denier/denier.go

```diff
@@ -39,15 +39,15 @@ type handler struct {
 func (h *handler) HandleCheckNothing(context.Context, *checknothing.Instance) (adapter.CheckResult, error) {
 	return adapter.CheckResult{
 		Status:        h.status,
-		ValidDuration: 1000 * time.Second,
+		ValidDuration: 5 * time.Second,
```

Comment:
Should these values be constants, annotated with comments on their setup?  Or maybe even config params for denier handlers?